wagtailseo/models.py

```diff
@@ -261,7 +261,10 @@ def seo_image_url(self) -> str:
         """
         base_url = utils.get_absolute_media_url(self.get_site())
         if self.seo_image:
-            return base_url + self.seo_image.get_rendition("original").url
+            seo_image_url = self.seo_image.get_rendition("original").url
+            if not seo_image_url.startswith('http'):
```

Comment:
We actually have a beefy regex to tell you whether a URL is absolute... you could replace this with
```python
if not PROTOCOL_RE.match(seo_image_url):
```

https://github.com/coderedcorp/wagtail-seo/blob/3fe003c41901c84778609f162185ed20a945c66c/wagtailseo/utils.py#L12-L13

Reply:
Cool, I've switched to using that.